Standards/scs-xxxx-v1-provider-network-standard.md

```diff
@@ -199,25 +201,42 @@ So, just specifying mandatory features rather than a certain set of extensions m
+
+The CSP **MUST** offer allocation of external IPv6 addresses to user projects.
+The CSP **SHOULD** offer allocation of external IPv4 addresses to user projects.
+
+### Standard Provider Network
+
+The CSP **MUST** provide every user project with a provider network that can route any external addresses that are allocated to the project.
+For external addresses from pool-allocated subnets, this requires support for dynamic routing.
+This provider network will in the following will be referred to as the _standard provider network_.
+
+The standard provider network **MUST** have the `is_default` flag set to `True`, and it **MUST** be the only provider network in the project with `is_default` set to `True`.
 To avoid ambiguity, the standard provider network **SHOULD** be the only provider network available to projects by default.
 
 The standard provider network **MUST** be an external network, allowing it to be used as external gateway by virtual routers.
-The standard provider network **MUST** have the `is_default` flag set to True, and it **MUST** be the only network to do so.
-The standard provider network **MAY** be a shared network, allowing direct attachment of virtual servers.
-If the standard provider network is a shared network, it **MUST** enable port security to prevent projects from interfering with each other.
+It **MAY** also be a shared network, allowing direct attachment of virtual servers.
+If the standard provider network is a shared network, then it **MUST** be configured allocate external addresses for directly attached servers and ports, and it **MUST** enable port security to prevent projects from interfering with each other.
+
+### IPv6 Allocation
+
+The CSP **MUST** provide a shared subnet pool for the allocation of at least one /64 prefix for external IPv6 addresses per project.
+This subnet pool **MUST** have the `is_default` flag set to `True`, and it **MUST** be the only shared IPv6 subnet pool in the project with `is_default` set to `True`.
+
+### IPv4 Allocation
 
-If CSPs offer public IP addresses at all, they **MUST** provide a shared subnet pool for the allocation of at least one public /64 IPv6 prefix per project.
-This subnet pool **MUST** have the `is_default` flag set to True, and it **MUST** be the only IPv6 subnet pool to do so.
-If CSPs offer public IP addresses, they **SHOULD** also offer public IPv4 addresses.
-If they do offer public IPv4 addresses, they **MUST** provide at least one public Floating IP per project, but **MAY** also provide a shared subnet pool for the allocation of public IPv4 prefixes to project networks.
-If CSPs offer a subnet pool for the allocation of public IPv4 prefixes, it **MUST** have the `is_default` flag set to True, and it **MUST** be the only IPv4 subnet pool to do so.
+If the CSP offers external IPv4 addresses, they **MUST** provide at least one external Floating IP per project that can be allocated from the standard provider network.
```

Comment:
Agree to name it external instead of pulic to include clouds which are only used internally / privately.